### PR TITLE
Make MQ plug-in V3 conformant

### LINKED ISSUE #60: Update the plug-in to be compatible with Zowe CLI V3
**Is your feature or enhancement request related to a problem or limitation? Please describe**

Zowe V3 is scheduled to be released at the end of March 2024. This plug-in should also be available for users upgrading to V3.

**Describe your enhancement idea**

The open source plug-ins should be updated to be compatible with Zowe CLI V3 (8.x.x) and should apply for, and achieve, V3 conformance.

**Describe alternatives you've considered**

**Provide any additional context**

### PR DESCRIPTION
**Pull request by zFernand0**
**What It Does**

Introduce changes needed to achieve V3 conformance for the MQ plug-in

**How to Test**

Go through the checklist 😋 

**Review Checklist**
I certify that I have:
- [x] tested my changes
- [x] added/updated automated tests
- [x] updated the changelog
- [x] followed the [contribution guidelines](https://github.com/zowe/zowe-cli/blob/master/CONTRIBUTING.md)

**Additional Comments**

### CODE REVIEW

**Review by gejohnston — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -94,21 +94,21 @@ Successfully validated.
...
+3.  Set the port number to your MQ port:
 
-**Tip:** For more information about the syntax, actions, and options, for a profiles create command, open Zowe CLI and issue the following command:
-
-```
-zowe profiles create mq-profile -h
-```
+    ```
+    zowe config set profiles.mq.properties.port <port number>
+    ```
```
Comment (anaxceron) [inline README.md:112]: Line 124: After you copy the file, what should you do to the file? It's not clear the way it's written. Should it be "To define access credentials to the server, copy the file named BBBB and rename it CCCC."?

File: README.md [hunk 2]
```diff
@@ -141,7 +141,7 @@ Any failures potentially indicate an issue with the set-up of the Rest API or co
     ```
     zowe plugins uninstall @zowe/mq-for-zowe-cli
     ```
-    
+
```
Comment (anaxceron) [inline README.md:144]: Line 145: We should name "the product"

**Review by anaxceron — APPROVED** [review]
Just needs a few edits. You should also check all the links to make sure they work. I think **Develop for Zowe CLI** no longer exists.

File: README.md (hunk 1 above)
Comment (zFernand0) [reply README.md:112]: The goal is to have them make an identical copy of the file with the new name.
This new file gets `git-ignored` (so `git` won't try to add it to the codebase).
This allows people to have their test-user and test-password in plain-text which we would use to submit z/OS requests 😋

**Pull request merged**